subnet/local/README.md

```diff
@@ -9,11 +9,19 @@ The purpose of this directory is for locally hosting subnet with multiple nodes.
 
 3. The `docker-compose.yml` provided is made for an example of 3 subnet nodes which can be increased as needed(requires new genesis file).
 
-4. `docker-compose up bootnode` start bootnode first to get bootnode URL and put in `.env` files. Other configs of the `genesis.json` and `.env` files are already pre-configured. 
+4. ~~`docker-compose up bootnode` start bootnode first to get bootnode URL and put in `.env` files. Other configs of the `genesis.json` and `.env` files are already pre-configured.~~ 
+Bootnode doesn't work as expected. It doesn't care about `enode_id` if just the IP+port is correct, the communication will work. So this step can be skipped.
```

Comment:
does it mean the step 4 is useless now?

Reply:
yeah, not sure it's always been this way? but it is convenient to just docker-compose up to start the network.